src/main/kotlin/edu/cwru/raycaster/Textures.kt

```diff
@@ -0,0 +1,20 @@
+package edu.cwru.raycaster
+
+import javafx.scene.image.Image
+import java.io.FileInputStream
+
+const val TEXTURE_PATH = "src/main/resources/edu/cwru/raycaster/textures/"
+
+enum class Texture(fileName: String) {
+    EAGLE("eagle.png"),
+    RED_BRICK("red_brick.png"),
+    PURPLE_STONE("purple_stone.png"),
+    GREY_STONE("grey_stone.png"),
+    BLUE_BRICK("blue_brick.png"),
+    MOSSY("mossy.png"),
+    WOOD("wood.png"),
+    COLOR_STONE("color_stone.png");
+
+    private val inputStream = FileInputStream(TEXTURE_PATH + fileName)
+    val image = Image(inputStream)
+}
```

Comment:
A possibility to avoid calling `.image`. I couldn't make it work with an actual enum though, so it's custom and wonky :(

```kotlin
class Test private constructor(fileName: String): Image(FileInputStream(TEXTURE_PATH + fileName)) {
    companion object {
        val EAGLE = Test("eagle.png")
        val RED_BRICK = Test("red_brick.png")
        val PURPLE_STONE = Test("purple_stone.png")
        val GREY_STONE = Test("grey_stone.png")
        val BLUE_BRICK = Test("blue_brick.png")
        val MOSSY = Test("mossy.png")
        val WOOD = Test("wood.png")
        val COLOR_STONE = Test("color_stone.png")
    }
}
```

Reply:
I think calling.image is probably simpler

Reply:
I know but it makes me sad